docs/source/installation/jupyter.rst

```diff
@@ -0,0 +1,101 @@
+Jupyter Notebooks
+=================
+
+
+Binder
+------
+
+`Binder <https://mybinder.readthedocs.io/en/latest/>`__ is an online
+platform that hosts shareable and customizable computing environments
+in the form of Jupyter notebooks. Manim ships with a built-in ``%%manim``
+Jupyter magic command which makes it easy to use in these notebooks.
+
+To see an example for such an environment, visit our interactive
+tutorial over at https://try.manim.community/.
+
+It is relatively straightforward to prepare your own notebooks in
+a way that allows them to be shared interactively via Binder as well:
+
+#. First, prepare a directory containing one or multiple notebooks
+   which you would like to share in an interactive environment. You
+   can create these notebooks by using Jupyter notebooks with a
+   local installation of Manim, or also by working in our pre-existing
+   `interactive tutorial environment <https://try.manim.community/>`__.
+#. In the same directory containing your notebooks, you need to add a
+   file named ``Dockerfile`` with the following content:
+
+   .. code-block:: dockerfile
+
+      FROM manimcommunity/manim:v0.9.0
```

Comment:
Stable might work, but I consider it bad practice as it might lead to code in notebooks breaking due to a library upgrade.

I wouldn't update this with every release, it is mentioned that the tag should be adapted.